### DISCUSSION TITLE
Must Play Services / Google Photos be uninstalled or is disabling them sufficient?

### DISCUSSION THREAD

**Discussion by ongymens** [lukaspieper/Gcam-Services-Provider#74] [Q&A] [answered]
Must Play Services be uninstalled or is it sufficient to have them disabled to install the app? If having them disabled is sufficient, what would happen if one was to enable Play Services again after installing the app? On a related note, must Google Photos be uninstalled or is it sufficient to have them disabled to install the "photos" version of the app? And as before: If having them disabled is sufficient, what would happen if one was to enable Google Photos again after installing the "photos" version of the app?

**Comment by lukaspieper** [comment, marked as answer]
Hi,

I think similar questions were answered all over this repository. I will keep it short:

you cannot **install** my app if you have official Play Services or microg on your device (even if disabled). Same for Google Photos, if you have Google Photos on your device you must go with the basic variant of my app but only if there is no Play Services or microg.
